scripts/analytics/src/index.ts

```diff
@@ -1,3 +1,14 @@
-const getPackageName = (): string => "CLI Analytics";
+import { Command } from "commander";
 
-export { getPackageName };
+import { setupCommands } from "./modules/cli/cli.js";
+
+const program = new Command();
+
+program
+	.name("CLI Analytics")
+	.description("CLI tool to collect repositories data and send analytics")
+	.version("1.0.0");
```

Comment:
We need a separate module for this logic, similar to server-application on backend as starting point. It should be called like AnalyticsCli or similar and it should init the application